safety_db/__init__.py

```diff
@@ -0,0 +1,37 @@
+import json
+import os
+__title__ = 'safety-db'
+__version__ = '2019.5.5'
+__author__ = 'Justin Womersley <support@pyup.io>'
+__copyright__ = '2016-2017 PyUp'
+__license__ = 'Attribution-NonCommercial-ShareAlike 4.0 International'
+__all__ = (
+    'INSECURE',
+    'INSECURE_FULL',
+)
+
+# Set default values in case there is an issue loading the data files.
+INSECURE = []
+INSECURE_FULL = []
+
+try:
+    # Get the data files from the package using pkg_resources so the package is zip safe
+    import pkg_resources
+    INSECURE = json.loads(pkg_resources.resource_string("safety_db", "data/insecure.json").decode(errors='ignore'))
```

Comment:
We have just a few data files, not sure it would be cool using `pkg_resource` just for that. This try/catch here seems very advanced for such a small package.

Reply:
This try/catch ensures the package works correctly if the package is being imported from an archive file and as a result isn't present on the filesystem.  It still falls back to reading it from the filesystem if pkg_resources isn't working.